### ISSUE TITLE
Change sql/createDatabase.sql script to avoid errors on execution

### ISSUE THREAD

**Issue by gdyrrahitis** [sextondjc/Drapper#6]
Should fix .sql script to avoid errors on execution.
Error messages from query execution:

```
Msg 5133, Level 16, State 1, Line 25
Directory lookup for the file "C:\Users\sexto\DrapperTesting.mdf" failed with the operating system error 2(The system cannot find the file specified.).
Msg 1802, Level 16, State 1, Line 25
CREATE DATABASE failed. Some file names listed could not be created. Check related errors.
Msg 5011, Level 14, State 5, Line 34
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 34
ALTER DATABASE statement failed.
Msg 911, Level 16, State 4, Line 38
Database 'DrapperTesting' does not exist. Make sure that the name is entered correctly.
Msg 5011, Level 14, State 5, Line 41
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 41
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 43
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 43
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 45
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 45
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 47
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 47
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 49
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 49
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 51
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 51
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 53
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 53
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 55
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 55
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 57
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 57
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 59
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 59
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 61
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 61
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 63
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 63
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 65
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 65
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 67
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 67
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 69
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 69
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 71
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 71
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 73
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 73
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 75
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 75
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 77
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 77
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 79
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 79
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 81
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 81
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 83
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 83
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 85
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 85
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 87
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 87
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 89
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 89
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 91
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 91
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 93
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 93
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 95
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 95
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 97
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 97
ALTER DATABASE statement failed.
Msg 911, Level 16, State 1, Line 99
Database 'DrapperTesting' does not exist. Make sure that the name is entered correctly.
Msg 5011, Level 14, State 5, Line 193
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 193
ALTER DATABASE statement failed.
Msg 5011, Level 14, State 5, Line 194
User does not have permission to alter database 'DrapperTesting', the database does not exist, or the database is not in a state that allows access checks.
Msg 5069, Level 16, State 1, Line 194
ALTER DATABASE statement failed.
Msg 911, Level 16, State 1, Line 197
Database 'DrapperTesting' does not exist. Make sure that the name is entered correctly.

```

**Comment by sextondjc** [comment]
Agreed. This has been a pain in my side for a while now.

**Comment by sextondjc** [comment]
I have a solution for this. Under  test at the moment.

**Comment by sextondjc** [comment]
Partially fixed in 1.1.0. SQL script no longer needed. 
Tests (currently) use AssemblyInitialize attribute to call Setup() method on EnvironmentHelper (new) which attempts to create a new database called Drapper and objects used by the tests. 

Could be more robust, but slightly better than what was available in 1.0.0. Not ready to close yet.

**Comment by sextondjc** [comment]
This is resolved in 1.1.0.
